commit d1f6583a5ff32711a1da1d4a13a29a5700f63504
Author: Ben Hutchings <ben@decadent.org.uk>
Date:   Thu Apr 2 11:43:33 2015 -0400

    mount.nfs.man, nfs.man: Update distinction between fstypes
    
    From: Ben Hutchings <ben@decadent.org.uk>
    
    this is a resync of the man page updates in the Debian
    package with mainline nfs-utils.
    
    Acked-By: J. Bruce Fields <bfields@fieldses.org>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/mount/mount.nfs.man b/utils/mount/mount.nfs.man
index 1a4561b..15a82d5 100644
--- a/utils/mount/mount.nfs.man
+++ b/utils/mount/mount.nfs.man
@@ -15,16 +15,20 @@ is meant to be used by the
 .BR mount (8)
 command for mounting NFS shares. This subcommand, however, can also be used as a standalone command with limited functionality.
 
-.BR mount.nfs4 
-is used for mounting NFSv4 file system, while 
-.BR mount.nfs 
-is used to mount NFS file systems versions 3 or 2.
 .I remotetarget 
 is a server share usually in the form of 
 .BR servername:/path/to/share.
 .I dir 
 is the directory on which the file system is to be mounted.
 
+Under Linux 2.6.32 and later kernel versions,
+.BR mount.nfs
+can mount all NFS file system versions.  Under earlier Linux kernel versions,
+.BR mount.nfs4
+must be used for mounting NFSv4 file systems while
+.BR mount.nfs
+must be used for NFSv3 and v2.
+
 .SH OPTIONS
 .TP
 .BI "\-r"
